builder/tart/step_run.go

```diff
@@ -201,41 +201,57 @@ func typeBootCommandOverVNC(
 	dialer := net.Dialer{}
 	netConn, err := dialer.DialContext(ctx, "tcp", fmt.Sprintf("%s:%s", vncHost, vncPort))
 	if err != nil {
-		err := fmt.Errorf("Failed to connect to the Tart's VNC server: %s", err)
+		err := fmt.Errorf("Failed to connect to Tart's VNC server: %s", err)
 		state.Put("error", err)
 		ui.Error(err.Error())
 
 		return false
 	}
 	defer netConn.Close()
 
+	serverMessageChannel := make(chan vnc.ServerMessage)
 	vncClient, err := vnc.Client(netConn, &vnc.ClientConfig{
 		Auth: []vnc.ClientAuth{
 			&vnc.PasswordAuth{Password: vncPassword},
 		},
+		ServerMessageCh: serverMessageChannel,
 	})
 	if err != nil {
-		err := fmt.Errorf("Failed to connect to the Tart's VNC server: %s", err)
+		err := fmt.Errorf("Failed to connect to Tart's VNC server: %s", err)
 		state.Put("error", err)
 		ui.Error(err.Error())
 
 		return false
 	}
 	defer vncClient.Close()
 
-	ui.Say("Connected to the VNC!")
+	ui.Say("Connected to VNC server!")
+
+	vncClient.SetEncodings([]vnc.Encoding{
+		&ZlibEncoding{},
+		&DesktopSizePseudoEncoding{},
+	})
+
+	vncDriver := newCustomDriver(vncClient, serverMessageChannel, config, ctx)
 
 	if config.VNCConfig.BootWait > 0 {
 		message := fmt.Sprintf("Waiting %v after the VM has booted...", config.VNCConfig.BootWait)
 		ui.Say(message)
 		time.Sleep(config.VNCConfig.BootWait)
+	} else {
+		ui.Say(fmt.Sprintf("Waiting for first frame..."))
```

Comment:
What is the use-case for this?

Can you just add `<wait '.*'>` to the `boot_commands` to achieve the same?